AngularLanguageService/AngularLanguageClient.cs

```diff
@@ -75,13 +77,13 @@ public async Task<Connection> ActivateAsync(CancellationToken token)
             process.StartInfo = info;
             process.ErrorDataReceived += (obj, data) => { outputPane.WriteAsync($"Error from node process: {data.Data}").Forget(); };
 
-            await outputPane.WriteAsync("Starting node process.");
+            outputPane.WriteAsync("Starting node process.").Forget();
```

Comment:
Can you extract all these instances of `outputPane.WriteAsync` to a helper method and document there that it's fire-and-forget to avoid awaiting the UI thread?

Reply:
Also, filed https://github.com/microsoft/vs-ng-language-service/issues/23 to clean up the output pane stuff a bit.

Reply:
Yep, done